SpiceQL/src/query.cpp

```diff
@@ -112,19 +128,22 @@ namespace SpiceQL {
 
     vector<string> outKernels = {};
     for (auto kernelList : files) {
-      outKernels.push_back(*(max_element(kernelList.begin(), kernelList.end())));
+      SPDLOG_TRACE("Max element: {}", *max_element(kernelList.begin(), kernelList.end(), fileNameComp));
```

Comment:
Looks like we're performing the search twice here.  Might make sense to save it off to a variable and log / push_back the variable instead of running max_element() twice.

That being said, I don't know how expensive max_element() is, so it might not matter?

Reply:
I can remove that line, was only used for my testing. Technically it would only run twice if log level was trace.